relay-general/src/protocol/session.rs

```diff
@@ -217,7 +217,7 @@ mod tests {
         let mut parsed = SessionUpdate::parse(json.as_bytes()).unwrap();
 
         // Sequence is defaulted to the current timestamp. Override for snapshot.
-        assert!((parsed.sequence - default_sequence()) <= 1);
+        assert!((default_sequence() - parsed.sequence) <= 1);
```

Comment:
Good catch! Since this is an unrelated change, can you please create a separate PR for this and add a description on the change?

Reply:
ok